content/en/docs/concepts/workloads/pods/sidecar-containers.md

```diff
@@ -17,7 +17,7 @@ security, or data synchronization, without directly altering the primary applica
 
 ## Enabling sidecar containers
 
-Starting with Kubernetes 1.28, a
+Enabled by default with Kubernetes 1.29, a
```

Comment:
nit: Can you reword to the sentence:
Starting with Kubernetes v1.29, the SidecarContainers [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled by default.
SidecarContainers allows you to specify a `restartPolicy` for containers listed ...